tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md

```diff
@@ -351,6 +351,8 @@ To start an individual test recording or playback, users will POST to a route on
 
 You will receive a recordingId in the reponse under header `x-recording-id`. This value should be included under header `x-recording-id` in all further requests.
 
+For playback, you will receive the location of the recording file (if it is on disk) in the header `x-recording-file-location`.
```

Comment:
If we're going to do this we might as well do it for both `Record/Start` and `Playback/Start` I think.